R/system.R

```diff
@@ -1402,6 +1410,11 @@ executeGoogleBigQuery <- function(project, query, destinationTable, pageSize = 1
   if(!requireNamespace("stringr")){stop("package stringr must be installed.")}
 
   token <- getGoogleTokenForBigQuery(tokenFileId)
+  # Begin Workaround ref: https://github.com/r-dbi/bigrquery/issues/395
+  # Remember original scipen
+  original_scipen = getOption("scipen")
+  # Temporary override it with 20 to workaround: Invalid value at 'start_index' (TYPE_UINT64), "1e+05" [invalid] error
+  options(scipen = 20)
```

Comment:
Make sure this reverting of scipen is done even in error case, using tryCatch.

Reply:
changed it to use tryCatch